README.md

```diff
@@ -14,6 +14,32 @@ vary across manufacturers, devices and android versions. This library was create
 to provide a consistent experience across all devices and users.
 
 
+## Installation
+
+Step 1. Add the JitPack repository to your build file
+
+```groovy
+allprojects {
+  repositories {
+    ...
+    maven { url 'https://jitpack.io' }
+  }
+}
+```
+
+Step 2. Add the dependency
+
+```groovy
+dependencies {
+    implementation 'com.github.prolificinteractive.geocoder:geocoder:0.1.0'
+    implementation 'com.github.prolificinteractive.geocoder:geocoder-googlemaps:0.1.0'
+    implementation 'com.github.prolificinteractive.geocoder:geocoder-okhttp:0.1.0'
+
+    // Optional
+    implementation 'com.github.prolificinteractive.geocoder:geocoder-openstreetmap:0.1.0'
+}
```

Comment:
should use add to the comment what the optional library provides?

Reply:
Agreed, but that should be in the usage section, imo